.cirrus.yml

```diff
@@ -13,7 +13,7 @@ airyx_task:
     SURGE_TOKEN: ENCRYPTED[b8c152a2c22ada1210122d08f0cb811b94d5c51380e13002debdd4f558be2b28e89af956b0980db4854ef0e4de850fcb]
   script:
     - pkg update && pkg install -y npm 
-    - npm install -g surge
```

Comment:
Because we already have `surge` in `package.json`, we can use the local package instead of installing it globally.